src/styles/blocks/card.css

```diff
@@ -0,0 +1,88 @@
+/* #region card */
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
```

Comment:
I think padding 32px 16px will be enough third one is unnecessary

Reply:
@ivan-kadykalo got you, found my mistake. 32px top, 16px left+riight, 16px bottom, if a delete last one , the bottom will be 32px also.